Tasks/google-play-release/GooglePlay.ts

```diff
@@ -195,106 +128,46 @@ async function run() {
             }
         }
 
+        let releaseNotes: googleutil.ReleaseNotes[];
         if (shouldAttachMetadata) {
             console.log(tl.loc('AttachingMetadataToRelease'));
             tl.debug(`Uploading metadata from ${metadataRootPath}`);
-            await addMetadata(edits, apkVersionCodes, metadataRootPath);
+            releaseNotes = await addMetadata(edits, apkVersionCodes, metadataRootPath);
+            requireTrackUpdate = true;
         } else if (changelogFile) {
-            tl.debug(`Upload the common change log ${changelogFile} to all versions`);
-            await uploadCommonChangeLog(edits, languageCode, changelogFile, apkVersionCodes);
+            tl.debug(`Uploading the common change log ${changelogFile} to all versions`);
+            const commonNotes = await getCommonReleaseNotes(languageCode, changelogFile);
```

Comment:
Just want to confirm that `uploadCommonChangeLog` has been replaced by `getCommonReleaseNotes` .

Reply:
Nvm, I see those are internal calls we created, not APIs.